sentry.server.config.ts

```diff
@@ -12,8 +12,10 @@ Sentry.init({
   dsn: process.env.SENTRY_DSN,
 
   // Adjust this value in production, or use tracesSampler for greater control
-  tracesSampleRate: 1,
+  tracesSampleRate: ["development", "heroku"].includes(process.env.NODE_ENV)
```

Comment:
I guess same comment as above where now we're juggling `NODE_ENV` _and_ `NEXT_PUBLIC_NODE_ENV`, but I'm not 100% clear on whether one inherently sets the other or if both need to be explicitly set.

Or is the logic that client-side code can use the Node-esque `process.env.NEXT_PUBLIC_NODE_ENV` and Next.js will cleverly shim the value into the client code, but the server code can just use `process.env.*` like it always has since it has access to env vars.

Reply:
I'm a little confused by this too honestly... https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables explains it but I'm not clear on why this would work.

Reply:
Well, so I do know that the idea is that `NEXT_PUBLIC_*` is a signal to the bundler that it's OK to include this var in front-end code (in other words, it does not contains secrets and can be public in the web page source).

However I mean I'm not clear on why `process.env.NODE_ENV` is an exception to this rule, is that what you mean in https://github.com/mozilla/blurts-server/pull/3138#discussion_r1238304742 @Vinnl ?

Reply:
Yeah, for some reason I can't find it in the docs, but I do recall `NODE_ENV` being the one exception to the `NEXT_PUBLIC_`. Which makes sense, since the prefix is there to avoid accidentally leaking secrets - but if you've got secrets in `NODE_ENV`, you'd have bigger problems :sweat_smile: